ui/src/app/utils/access-utils.tsx

```diff
@@ -113,6 +114,8 @@ export const redirectToRas = (openInNewTab: boolean = true): void => {
 interface AccessModuleConfig {
   moduleName: AccessModule;
   isEnabledInEnvironment: boolean; // either true or dependent on a feature flag
+  isRequiredForRT: (Profile) => boolean;
```

Comment:
I still think this logic should be kept separate from configuration. Configuration is driven more by environment. This decision is user's profile driven, it should be happening in a separate function